Szymon/Zadanie7/src/test/java/Szymon7/page/WelcomePage.java

```diff
@@ -1,12 +1,11 @@
 package Szymon7.page;
 
+import Szymon7.User;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.FindBy;
 
-public class WelcomePage extends LeftNavigationMenu {
+public class WelcomePage extends CustomerLoginPage {
 
-    public WelcomePage(WebDriver driver, String url) {
+    public WelcomePage(WebDriver driver, String url, User user) {
```

Comment:
A co sie dzieje z tym userem?
Usunac.

Reply:
No potrzebowałem go podać do scenariusza FullRegisterUserScenario a wówczas wołało o kolejne zmiany i tak mi intelij podpowiedział. Jak nie tak to jak?